src/app/dashboard/dashboard.component.ts

```diff
@@ -241,8 +244,15 @@ export class HeaderComponent implements OnInit, OnDestroy {
       });
   }
 
-  navigateToPage(abbreviation) {
-    this.route.navigateByUrl(this.dispLanguageCode + '/' + abbreviation);
+  navigateToPage(resource: Resource): void {
+    const abbreviation = resource.abbreviation;
+    const urlResourceType = resource.resourceType === 'tract' ? 't' : 'c';
```

Comment:
Let me find the message, but we settled on `v1` and `v2`

Reply:
Sorry. Thank you for reminding me.